### PR TITLE
feat: add support for metrics_jitter

### PR DESCRIPTION
**Pull request by ivarconr**
fixes: #108

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ Keep-Alive: timeout=5
...
 | metricsInterval           | `UNLEASH_METRICS_INTERVAL`       | 30000           |    no    | How often the proxy should send usage metrics back to Unleash, defined in ms.                                                                                                                                                                                                                                                                                                                                                                                            |
+| metricsJitter             | `UNLEASH_METRICS_JITTER`         | 0               |    no    | Adds jitter to the metrics interval to avoid multiple instances sending metrics at the same time, defined in ms.                                                                                                                                                                                                                                                                                                                                                                                            |
```
Comment (sighphyre) [inline README.md:118]: Was just about to ask for an update to the README. Refreshed the page and suddenly it appears! Magic

**Review by sighphyre — APPROVED** [review]
LGTM

File: package.json [hunk 2]
```diff
@@ -44,7 +44,7 @@
     "express": "^4.18.2",
```
[the remark this answers is not in the document]
Comment (ivarconr) [reply package.json:44]: will upgrade in a separate fix.

[the remark this answers is not in the document]
Comment (ivarconr) [reply package.json:44]: @sonatype-lift ignore

**Pull request merged**